target_postgres/__init__.py

```diff
@@ -30,7 +30,15 @@ def flush_streams(streams, target, force=False):
         if force or stream_buffer.buffer_full:
             flush_stream(target, stream_buffer)
 
-def line_handler(streams, target, max_batch_rows, max_batch_size, line):
+def report_invalid_records(streams):
+    for stream_buffer in streams.values():
+        if stream_buffer.peek_invalid_records():
+            LOGGER.error("Invalid records detected for stream {}: {}".format(
```

Comment:
Since this will run every time, we may want to warn if less than invalid_records_threshold and error if at or greater than invalid_records_threshold